modules/ROOT/pages/cassandra/cassandra-get-started.adoc

```diff
@@ -0,0 +1,37 @@
+= Cassandra Connector - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Support Category: Select
+
+Cassandra Connector v1.0
+
+The Anypoint CassandraDB connector creates connectivity between Cassandra databases and third-party applications, systems, services such as SAP, PeopleSoft, or Workday as well as other databases across the enterprise. Apache Cassandra is an open source distributed NoSQL database intended to manage large amounts of product, customer, and employee data. 
+
+== Audience
+
+ * New user -- Read this document and configure your Mule application in xref:cassandra/cassandra-design-center.adoc[Design Center].   
+* Current Mule user -- Read this document and configure your Mule application in xref:cassandra/cassandra-design-center.adoc[Design Center] or xref:cassandra/cassandra-studio.adoc[Anypoint Studio]. 
+The xref:cassandra/cassandra-config-topics.adoc[Configuration Topics] help you extend your understanding of Salesforce actions.
```

Comment:
It is not clear what "Salesforce acctions" are.

Reply:
We probably want to be a bit more specific here, there are a lot of SFDC actions: https://developer.salesforce.com/docs/atlas.en-us.api_action.meta/api_action/actions_intro_list.htm#

Why is line 16 under Audience and only for current Mule user? What is this section for? Are we saying only these three types of people should expect to use Cassandra connector? Are we claiming the listed docs are the only ones needed for user of type listed? What problem is this section solving?

Reply:
This is template feedback which I think Diane and Mysti will forward to Meera.

Reply:
All good generic template questions to pass along to Meera.